website/sidebars.json

```diff
@@ -63,6 +63,9 @@
     "Cumulus Development": [
       "adding-a-task",
       "docs-how-to"
+    ],
```

Comment:
We *probably* should include something here indicating which version threshold this is applicable to.    Otherwise, if we continue using this pattern the sidebar could become confusing.  

Alternately, I suppose we could explicitly include a reference in the CHANGELOG/Release post and call that out *in* the doc itself 🤔

Reply:
https://github.com/nasa/cumulus/pull/1863/commits/90d61f20f71e08940a1e2b9a104f83e8f3bc4e8e#diff-190568db1790300e626312496836b296R9